### ISSUE TITLE
[BUG] Census / Reference report mismatches - Multiple 'Founds' AND 'References'

### ISSUE THREAD

**Issue by bmurcutt** [ShammyLevva/FTAnalyzer#100]
7.3.7.0 (beta)

1 EVEN Bunt Lodge, Nether Street
2 TYPE Census 1841
2 DATE 6 Jun 1841
2 PLAC , Finchley, Greater London (Middlesex), England
2 SOUR @S46@
3 PAGE Bunt Lodge, Nether Street, Finchley, Barnet, London, Englan
4 CONC d, folio 3/16, page 24, William Morcot; digital images, \
4 CONC i FindMyPast.co.uk\i0  (https://search.findmypast.co.uk/sea
4 CONC rch-world-Records/1841-england-wales-and-scotland-censu
4 CONC s : accessed 22 Sep 2009); citing PRO HO/107/RG HO107
3 QUAY 1
3 NOTE Not sure ... matches age, William Morcot not found in othe
4 CONC r Census or BMD indexes, so could be a reasonable mis-spell
4 CONC ing.
3 DATA
4 TEXT
5 CONT First name(s) Last name Gender Age Birth year Birth place
5 CONT James Block Male 45 1796 Middlesex, England
5 CONT Susan Block Female 40 1801 Scotland
5 CONT Eliza Block Female 15 1826 Middlesex, England
5 CONT Henrietta Wilson Female 35 1806 Scotland
5 CONT Charles Gibbs Male 35 1806 -
5 CONT Ann Corn Female 35 1806 Scotland
5 CONT Mary Muddle Female 20 1821 -
5 CONT Caroline Frayer Female 25 1816 -
5 CONT Beartrice Thomson Female 30 1811 Scotland
5 CONT Harriott Cramp Female 20 1821 -
5 CONT William Morcot Male 20 1821 -
5 CONT Benjamin Chalk Male 15 1826 Middlesex, England
5 CONT
5 CONT Street Nether Street Bunt Lodge???
5 CONT Parish or township Finchley
5 CONT City or borough -
5 CONT Town -
5 CONT County Middlesex
5 CONT Country England
5 CONT Registration district Barnet
5 CONT Archive reference HO107
5 CONT Piece number 663
5 CONT Book number 3
5 CONT Folio number 16
5 CONT Page number 24
5 CONT Record set 1841 England, Wales & Scotland Census
5 CONT Category Census, land & surveys
5 CONT Record collection Census
5 CONT Collections from Great Britain
3 OBJE
4 FORM jpg
4 FILE C:\Users\BrianM\Documents\Family Tree\Records\Murcutt records\William Murcutt 1819 (Morcot) - Census 1841.jpg
4 TITL (Possible) 1841 Census entry for William Murcutt
4 _DATE 1841
4 _SCBK Y
4 _PRIM Y
4 _TYPE PHOTO

1 EVEN 9 Bartholomew Place
2 TYPE Census 1851
2 DATE 30 Mar 1851
2 PLAC , Bethnal Green, Greater London (Middlesex), England
2 SOUR @S47@
3 PAGE (https://search.findmypast.co.uk/search-world-Records/1851-
4 CONC england-wales-and-scotland-census : accessed 16 Dec 2009)
4 CONC , entry for William Murcutt, Bethnal Green
3 QUAY 2
3 DATA
4 TEXT
5 CONT First name(s) Last name Relationship Marital status Gende
5 CONC r Age Birth year Occupation Birth place
5 CONT William Murcutt Head Married Male 32 1819 Labourer Stratfor
5 CONC d, Essex, England
5 CONT Caroline Murcutt Wife Married Female 29 1822 - Whitechapel
5 CONC , Middlesex, England
5 CONT Henry Murcutt Son - Male 3 1848 - Bethnal Green, Middlesex
5 CONC , England
5 CONT William Murcutt Son - Male 1 1850 - Bethnal Green, Middlese
5 CONC x, England
5 CONT James Jarry Lodger Unmarried Male 23 1828 Tailor Bethnal Gr
5 CONC een, Middlesex, England
5 CONT
5 CONT Street Bartholomew Place
5 CONT Town -
5 CONT Parish Bethnal Green
5 CONT City The Tower Hamlets
5 CONT County Middlesex
5 CONT Hamlet -
5 CONT Parliamentary borough The Tower Hamlets
5 CONT Municipal ward -
5 CONT Registration district Bethnal Green
5 CONT Archive reference HO107
5 CONT Piece number 1541
5 CONT Folio 119
5 CONT Page 15
5 CONT Record set 1851 England, Wales & Scotland Census
5 CONT Category Census, land & surveys
5 CONT Subcategory Census
5 CONT Collections from Great Britain
3 OBJE
4 FORM jpg
4 FILE C:\Users\BrianM\Documents\Family Tree\Records\Murcutt records\William Murcutt 1819 - Census 1851.jpg
4 TITL 1851 Census entry William Murcutt and family
4 _DATE 1851
4 _SCBK Y
4 _PRIM Y
4 _TYPE PHOTO

The predominant issue with this one is that FTAnalyzer appears to have treated an 1851 Census reference as an 1841!  It looks like in the ‘CensusReferences.cs’ code you look at the piece number to determine whether it is 1841 or 1851, but somewhere it must be going amiss …

Found entries (because of family groups):

F0945 | I3321 | Finchley, Greater London (Middlesex),   England | William Murcutt | 21 |   | 5 NOV 1819 | Mile End Old Town, Greater London   (Middlesex), England | 26 JAN 1886 | Bethnal Green Workhouse, Bethnal   Green, Greater London (Middlesex), England | Husband | Direct Ancestor | 2nd great grandfather | UK Census 1841

F0971 | I3321 | Finchley, Greater London (Middlesex),   England | William Murcutt | 21 |   | 5 NOV 1819 | Mile End Old Town, Greater London   (Middlesex), England | 26 JAN 1886 | Bethnal Green Workhouse, Bethnal   Green, Greater London (Middlesex), England | Child | Direct Ancestor | 2nd great grandfather | UK Census 1841

Good Reference entries:

  | I3321 | William | Murcutt | **Census (FTAnalyzer)** | UNKNOWN | Murcutt | Direct Ancestor | 2nd great grandfather | 5 NOV 1819 | England and Wales | **Fact created by FTAnalyzer after   finding census ref: RG8 Piece 1541 Folio 119 Page 15 in the notes for this   individual** | Piece: 1541, Folio: 119, Page: 15 | Unknown | FALSE |  

  | I3321 | William | Murcutt | Census | 6 JUN 1841 | Murcutt | Direct Ancestor | 2nd great grandfather | 5 NOV 1819 | Finchley, Greater London (Middlesex),   England | Bunt Lodge, Nether Street | 6 JUN 1841 |   | 21 | FALSE | 1841 England, Wales & Scotland   Census. Digital images.

I also note that the ‘valid’ 1841 entry does not generate the Census Reference information – though it is available in the GEDCOM, and has been determined to be ‘good’.  And yes, it is one of the ‘Failed to add’ results in the Lost Cousins load!

**Comment by ShammyLevva** [comment]
1851 and 1841 references are identical as they simply incremented the piece number by 1 from the end of the 1841 to the start of 1851 census when they indexed them. So the issue may be in the piece number.

Looking at the GEDCOM I see

5 CONT Archive reference HO107
5 CONT Piece number 1541
5 CONT Folio 119
5 CONT Page 15
5 CONT Record set 1851 England, Wales & Scotland Census

Shows up for me as 1851. So I'm not sure why you are getting it as 1841. 

As you say this may be to do with Family Groups but without the Family Group GEDCOM I cannot replicate the issue.

**Comment by bmurcutt** [comment]
I'll have another look at this later and perhaps try and create a small GEDCOM that recreates the problem.

**Comment by bmurcutt** [comment]
Hi Alexander.

 

Attached is a GEDCOM for Issue 100.

 

Person of interest is William Sprague (i3321).

 

I don’t know what I was looking at before – I was sure it was all 1841 details!!??! – but the 1851 reference is handled correctly in this extract (FTA didn’t ‘create’ an 1841 reference using the 1851 details like it looked like it did before).

 

Last time, I extracted this clip from the Found list:

 

I3321 | William | Murcutt | Census (FTAnalyzer) | UNKNOWN | Murcutt | Direct Ancestor | 2nd great grandfather | 5 NOV 1819 | England and Wales | Fact created by FTAnalyzer after finding census ref: RG8 Piece 1541 Folio 119 Page 15 in the notes for this individual | Piece: 1541, Folio: 119, Page: 15 | Unknown | FALSE |  

 

FTA isn’t auto-generating anything now, for 1841 or 1851.  Can’t see how I imagined it, but I can’t reproduce it now!!

 

I even reran my full file – but same (nothing) there now!

 

However you will note that the ‘valid’ 1841 entry for William does not generate a census reference (at all – not even a partial!), but it is flagged as ‘good’.  The reference:

 

5 CONT Archive reference HO107
5 CONT Piece number 663
5 CONT Book number 3
5 CONT Folio number 16
5 CONT Page number 24

 

Looks valid to me (EW_CENSUS_1841_51_PATTERN2).

 

Regards,

 

Brian
